collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
 ---
 layout:        infographic
-title:         "What is Fit for 55"
+title:         "What is 'Fit for 55'"
 slug:          "fit-for-55"
 redirect_from: "/fit-for-55"
-published:     2022-07-26
+published:     2022-11-30
 weight:        93
 tags-scopes:   [ eu ]
 tags-topics:   [ policies, economy ]
-caption:       "Fit for 55 is a package of legislation proposals designed by the European Commission to cut the EU's net greenhouse gas emissions by 55% below 1990 levels by 2030. It is a key step for the EU on its way to reach its legally binding target – net zero by 2050."
+caption:       "'Fit for 55' is a package of legislation proposals designed by the European Commission to cut the EU's net greenhouse gas emissions by 55% below 1990 levels by 2030. It is a key step for the EU on its way to reach its legally binding target – net zero by 2050."
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/fit-for-55" %}
+{% comment %}
+{% include preview-box.html
+    title="What are the individual measures in the 'Fit for 55' package?"
+    text="In the related infographic, we briefly discuss the content of the individual legislative proposals in the package."
+    slug="fit-for-55-opatreni"
+%}
+{% endcomment %}
+
+['Fit for 55'](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A52021DC0550), a package of legislative proposals that promotes a **reduction of greenhouse gas emissions by 55% by 2030 compared to 1990**, was presented by the European Commission in July 2021. The package includes proposals amending existing directives or regulations and entirely new legislative proposals, focusing on emissions trading (EU ETS), carbon offsetting at borders, energy and fuels, and land use and forestry. Last but not least, it brings in supporting measures to make the transition more socially just. The proposed measures should bring new opportunities for investment, innovation and job creation[^fit-for-55].
+
+The package is currently at the stage of legislative proposals by the European Commission, which are being further discussed in the European Parliament and the Council of the European Union, i.e. with the participation of ministers of the governments of the Member States and national working groups. In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu], which it will assume in the second half of 2022, is crucial and will thus have the opportunity to influence the direction of the negotiations. It is expected that negotiations on the final form of the legislation could last until 2024.[^fit-for-55-q-a]
```

Comment:
I suggest we adjust to:

In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu] in the second half of 2022 is crucial, giving the Czech Republic the opportunity to influence the direction of the negotiations.

Reply:
Before I made any correction, I also noticed that the second comma after the reference `[^presidency-eu]` also looked weird.

I updated the text as per your comment.